app/services/deployment/project_instance_events.rb

```diff
@@ -25,18 +25,17 @@ def create_event(event)
       comment = Notifications::Comment.new(@project_instance)
       text = COMMENT_MESSAGES.fetch(event).call(comment)
 
-      pull_request.update_info_comment(text)
-
+      @project_instance.update!(deployment_status: event)
+      pull_request.update_info_comment(text) if @project.integration_type == ProjectsConstants::Providers::GITHUB
       Slack::Notificator.new(@project).send_message(text) if SLACK_EVENTS.include?(event)
     end
 
     private
 
     def pull_request
       @_pull_request ||= begin
-        pr_params = [@project.integration_id, @project_instance.repo_path, @project_instance.pull_request_number]
-
-        return unless pr_params.all?(&:present?)
+        pr_params = [@project.integration_id, @project_instance.attached_repo_path, @project_instance.attached_pull_request_number]
+        return if pr_params.any?(&:blank?)
```

Comment:
It'll just fail with 'no method error' in this case. I think it's better to extract to method with integration type check

Reply:
there was the pull_request check for existence, so i'll just regroup all checks